src/utils/storage.ts

```diff
@@ -35,3 +35,28 @@ export async function setStoreData(
     console.log(error.message)
   }
 }
+
+const NEW_EXPOSURES = "NEW_EXPOSURES"
+export async function getNewExposuresData(): Promise<boolean> {
```

Comment:
I may be a bit confused here, but why do we need this new entry in the async store?  Dont we already have the exposure info data? that is we either have a new exposure or we dont? and saving this boolean separately introduces multiple sources of truth?

Reply:
This is the first part of the work we introduced in #15 in order to have more type safety around what data we are saving to the device. Since the time that the exposure was detected and the time that the user saw that they have a new exposure are different, we have to track the status of whether or not a user has seen their most recent exposure separately. Matt had suggested that this should persist if a user force quits without acknowledging the notification, which is why we are keeping it in the async store.